Chaincase.UI/Components/SelectableCoin.razor

```diff
@@ -1,10 +1,14 @@
 ﻿@using Chaincase.Common
-@inject Global Global  
+@inject Global Global
 @inherits ReactiveComponentBase<CoinViewModel>
 
-<ion-label>@(PrivacyLabel(ViewModel.AnonymitySet)) @ViewModel.Amount </ion-label>
-<IonCheckbox slot="start"
-                @bind-Value="ViewModel.IsSelected" />
+<IonItem>
+    <IonCheckbox slot="start"
+                 @bind-Value="ViewModel.IsSelected" />
+    <p class="ion-float-left">@ViewModel.Label</p>
+    <p class="ion-float-right">@ViewModel.Amount</p>
+    <p class="ion-float-right">@PrivacyLabel(ViewModel.AnonymitySet)</p>
```

Comment:
ion-label is more accessable for inputs cause it's associated with the checkbox for screen readers

Reply:
I sent bitcoin to a blind man in a McDonalds once. He used Blockchain wallet clearly because it was accessible. Let's be accessible.

Reply:
also this doubles IonItem which already exists in SelectCoinsPage. We should refactor to use that page.

Reply:
All about it 💪